helpers/queryType.js

```diff
@@ -1,3 +1,11 @@
-const queryType = (httpVerb) => `.method("${httpVerb.toUpperCase()}", null)`;
+const queryType = (httpVerb) => {
+  // For GET and HEAD requests return `null` body since okttp doesn't like empty requests.
+  let body = "null";
+  if (httpVerb != "get" && httpVerb != "head") {
+    // For other requests such as PUT or POST if the body is empty `null` is not a valid body.
+    body = `RequestBody.create("", null)`;
+  }
+  return `.method("${httpVerb.toUpperCase()}", ${body})`;
+};
```

Comment:
Also, what about 
```suggestion
const queryType = (httpVerb) => {
  // For GET and HEAD requests return `null` body since okttp doesn't like empty requests.
  let body = "null";
  if (httpVerb != "get" && httpVerb != "head") {
    // For other requests such as PUT or POST if the body is empty `null` is not a valid body.
    return `.method("${httpVerb.toUpperCase()}", RequestBody.create("", null))`;
  } else {
    return `.method("${httpVerb.toUpperCase()}")`;
  }
};
```

Reply:
Tried it and it doesn't work :(